src/NuGet.Core/NuGet.CommandLine.XPlat/Native/NativeMethods.Darwin.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace NuGet.CommandLine.XPlat.Native
```

Comment:
Actually we tend to limit namespace levels. See for instance NuGet.Protocol.Core.v3 classes.